src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -652,6 +659,102 @@ public class LibraryServiceTest {
 
   @Test
   @SuppressWarnings("all")
+  public void streamShelvesTest() {
+    StreamShelvesResponse expectedResponse = StreamShelvesResponse.newBuilder().build();
+    List<GeneratedMessageV3> expectedResponses = new ArrayList<>();
+    expectedResponses.add(expectedResponse);
+    mockLibraryService.setResponses(expectedResponses);
+    StreamShelvesRequest request = StreamShelvesRequest.newBuilder().build();
+
+    MockStreamObserver<StreamShelvesResponse> responseObserver = new MockStreamObserver<>();
+
+    StreamingApiCallable<StreamShelvesRequest, StreamShelvesResponse> callable =
+        api.streamShelvesCallable();
+    callable.serverStreamingCall(request, responseObserver);
+
+    try {
+      List<StreamShelvesResponse> actualResponses = responseObserver.future.get();
+      Assert.assertEquals(1, actualResponses.size());
+      Assert.assertEquals(expectedResponse, actualResponses.get(0));
+      Assert.assertEquals(0, responseObserver.errors.size());
+    } catch (Exception e) {
+      Assert.fail();
+    }
```

Comment:
Why do we need a try/catch here? Won't an exception cause the test to fail anyway?